### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SabirAmeen**
Hi @vijaydevin
Please find the implementation for the header and content of gizmodo.

### CODE REVIEW

**Comment by SabirAmeen** [comment]
@GayathriMohan

File: index.html [hunk 1]
```diff
@@ -5,56 +5,142 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+    <link rel="icon" type="image/png" href="images/favicon.png">
 </head>
+<script id="scroller-content" type="text/x-jQuery-tmpl">
+    <article class="article-content">
+        <figure>
+            <a href="">
+                <img src="${articleImage}" alt="">
```
Comment (GayathriMohan) [inline index.html]: alt attr should be filled

File: sass/Partials/_content.scss [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            width: 95%;
+            padding: 0.5rem;
+            article {
+                width: 48%;
+                display: inline-block;
+                vertical-align: top;
+                margin-bottom: 10px;
+            }
+            img {
+                width: 100%;
```
Comment (GayathriMohan) [inline sass/Partials/_content.scss]: use mixin

File: sass/style.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@import "compass/reset";
+@import "compass";
+@import "Modules/mixin";
+@import "Modules/variables";
+@import "Partials/common";
+// header
```
Comment (GayathriMohan) [inline sass/style.scss]: dont push commented code

**Pull request closed without merging**